### PR TITLE
Add desktop images and styling.

### PR DESCRIPTION
**Pull request by AlyAtalla**
Added required fields, images, and styles for the desktop version.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -541,3 +545,231 @@ input::placeholder {
...
+    white-space: nowrap;
+  }
+
+  .cards {
+    display: grid;
+    grid-template-columns: repeat(3, 0.21fr);
+    padding: 0.1px;
+    padding-top: 5em;
+    gap: 25px;
+  }
```
Comment (AlphaNtihinduka) [inline style.css]: - [x] I have noticed that the cards section may be among one causing the horizontal scroll in your desktop version portifolio. Remember, The experience of scrolling horizontally can makes the screen judder and create a headache inducing feeling. Therefore I suggest that you fix horizontal scroll effect in all the desktop screen sizes.
Your design 😎 . Please disregard the background colors.
image

File: style.css [hunk 2]
```diff
@@ -541,3 +545,231 @@ input::placeholder {
...
+    margin-right: 27rem;
+    font-size: 31px;
+    margin-bottom: 49rem;
+  }
+
+  .form {
+    width: 100%;
+    min-width: 300px;
+    margin-bottom: 49rem;
+  }
```
Comment (AlphaNtihinduka) [inline style.css]: - [ ] I have noticed that the form area may be among one causing the horizontal scroll in your desktop version portifolio. Remember, The experience of scrolling horizontally can makes the screen judder and create a headache inducing feeling. Therefore I suggest that you fix horizontal scroll effect in all the desktop screen sizes.
Your design 😎 . Please disregard the background colors.
image

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hello @AlyAtalla 🙋‍♂️
Good job working on your contact form. You have done the design to the best expectation and incredibly well. 👏  

You still need to implement few on the project but you are almost there!!! :watch: 

TO HIGHLIGHT:
- Desktop size starts at 768px ✔️ 
- Well-structured files :heavy_check_mark: 

# STATUS: Required Changes :recycle: 

close

# Required Changes :recycle: 
_- Kindly refer to the inline review comment ☺️._
- [x] Please try as much as possible to fix the horizontal scroll within your portifolio to ensure that the design is professional

## Optional suggestions
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @AlyAtalla   👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job on implementing the project requirements! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉

## Highlights
✔ Made suggested changes 👍 
✔ Descriptive Pull Request title and summary. 👍  
✔ Good use of GitHub flow 👍 
### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/enus/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**